accounts/accounts.go

```diff
@@ -14,17 +14,20 @@ var (
 
 // Action : A Buy or Sell request that can expire
 type Action struct {
-	time      time.Time
-	stock     string
-	units     uint
-	unitPrice currency.Currency
+	Time      time.Time
+	Stock     string
+	Units     uint
+	UnitPrice currency.Currency
 }
 
+// Portfolio : User's stock holdings, stockName -> quantity
+type Portfolio map[string]uint
```

Comment:
IMO, makes it easier to make new instances instead of remembering the two types in the map.